node.go

```diff
@@ -246,6 +250,30 @@ func (n *node) collectedCount() (count int64) {
 	return
 }
 
+func (n *node) emittedCount() (count int64) {
+	for _, out := range n.outs {
+		count += out.emittedCount()
```

Comment:
This needs to read `count += out.collectedCount()` see comment in node.collectedCount